doc/pages/Packaging.md

```diff
@@ -130,7 +130,8 @@ than a single element: `maintainer`, `author`, `homepage`, `bug-reports`,
 `license` and `depends`.
 
 For the list of available fields and specification of the format, see the
-[OPAM manual](Manual.html#opam).
+[OPAM manual](Manual.html#opam). Also consider using the `opam lint` command to
+check the validity and quality of your `opam` file.
```

Comment:
This is indeed very worth mentioning! Note also that `opam pin edit` runs a lint check already.